server/package.json

```diff
@@ -12,16 +12,12 @@
     "dev:test": "NODE_ENV=test PORT=$npm_package_config_test_port babel-node src",
     "dev:test:kill": "lsof -t -i:$npm_package_config_test_port | xargs kill -15 | xargs kill -9",
     "debug": "nodemon -w src src/index.js --exec 'babel-node --inspect'",
-
     "test": "yarn test:api",
     "test:api": "yarn test:dredd; yarn dev:test:kill",
     "test:dredd": "babel-node node_modules/.bin/dredd",
-
     "lint": "eslint src",
-
     "build": "babel src -d build",
     "start": "node build",
-
```

Comment:
@DDKatch I was thinking about living these new lines for better groups separation, thoughts?